config/initializers/health_checks.rb

```diff
@@ -0,0 +1,24 @@
+# okcomputer is used for health checks
+
+username = ENV["HEALTHCHECK_HTTP_BASIC_AUTH_USERNAME"] || ENV["HTTP_BASIC_AUTH_USERNAME"]
+password = ENV["HEALTHCHECK_HTTP_BASIC_AUTH_PASSWORD"] || ENV["HTTP_BASIC_AUTH_PASSWORD"]
```

Comment:
Should these be in `Rails.application.secrets` - we are running all other env vars through there, mostly so there is a single place where config happens

Reply:
@eoinkelly I was following what was in a client project, which gets it's basic auth credentials directly from ENV - in the rails template this is in app/middleware/http_basic_auth_middleware.rb

It could be confusing if I source these creds from secrets.yml when the basic auth middleware doesn't get them from the same place?

Reply:
Hmmm I forgot that the middleware was using ENV vars directly. I agree we should be consistent but maybe the right move is to update the middleware to use Rails secrets too? I'm happy to make that change